tests/test_datastore.py

```diff
@@ -31,35 +33,35 @@ def mock_requests_get(url):
 
 class DatastoreTests(unittest.TestCase):
     def test_get_archive_dir(self):
-        result = fv3config.get_cache_dir()
+        result = fv3config.cache_location.get_cache_dir()
         assert isinstance(result, str)
         assert os.path.isabs(result)
 
     def test_set_then_get_archive_dir(self):
         with tempfile.TemporaryDirectory() as tempdir:
-            original = fv3config.get_cache_dir()
+            original = fv3config.cache_location.get_cache_dir()
```

Comment:
FYI changes like this were done automatically by PyCharm's refactor operation. I'm leaving it because it's fine to be explicit about where the function lives in the test files, but it does still exist at the top level and that's where it should be used in user code.